### ISSUE TITLE
Udi Weinstein didn't recieve the confirmation e-mail (30/6, ~4PM)

### ISSUE THREAD

**Issue by assafharel** [PeerSay/Atlas#107]
Hi Pavel,

We demoed to a potential user yesterday around 4PM.
He signed up but never got the confirmation e-mail.
Maybe this relates to the issues Tal was experiencing on the to Logentries crash report earlier today.

Assaf

**Comment by Pavek** [comment]
According to Sendbox, the email to udiw@mellanox.com was *delivered* at 4:20:
http://gyazo.com/b13157bddf72eeea197689b315da87b5

Please ask him to check Spam folder.

**Comment by assafharel** [comment]
We did ask him.
He didn't receive it.
Any other idea?

**Comment by Pavek** [comment]
Here's the answer from Sendgrid FAQ:
https://support.sendgrid.com/hc/en-us#collapse9

**Why does my account show a message as “delivered” when the recipient says that they haven’t received it?**
One of our most commonly-used reporting tools is the Email Activity page. However, there may be times when your email activity log shows a message as “delivered” but the recipient reports that they did not receive it. Whenever we receive a “250 OK” response from a recipient mail server it means that they have accepted the message handoff, and thus the responsibility, of delivering the message to its intended recipient. Unfortunately, this is also where message tracking capabilities end. Sometimes a recipient mail server will hold an accepted message in an inbound filter queue (in order to ensure that nothing within the message is harmful) before sending it to the recipient. Other times the message can simply end up in the spam folder and go overlooked.


Simply put, organization's mail server accepted it, but could not deliver to inbox for some reason. Probably it's worth to ask their email admin.

**Comment by assafharel** [comment]
OK, closing this issue
